src/screens/Dapp/QRCodeScanner/view.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Text } from 'react-native';
+import QRCodeScanner from 'react-native-qrcode-scanner';
+import styles from './styles';
+import Colors from '../../../global/colors';
+
+const View = ({ onRead }: Object) => (
+  <QRCodeScanner
+    onRead={onRead}
+    topContent={
+      <Text style={styles.centerText}>Scan barcode to connect to PC</Text>
+    }
+  />
+);
+
+View.navigatorButtons = {
+  leftButtons: [
+    {
+      id: 'cancel',
+      title: 'Cancel',
```

Comment:
I think we chose all visible text from the translation file.

Reply:
yes, i will change this defend on translation file